batchflow/models/tf/inception_base.py

```diff
@@ -32,6 +32,19 @@ class Inception(TFModel):
             value : dict
                 specific parameters (e.g. filters)
     """
+
+    def build_config(self, names=None):
```

Comment:
Add docstring

Reply:
Is that really needed? There are no docstrings for `build_config` in any of the models presented.